qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -0,0 +1,353 @@
+
+    .. math::
+
+        &&\hat A = \sum_{\alpha \leq 2N_\mathrm{docc}} \langle \alpha \vert \hat{\mathcal{A}}
+        \vert \alpha \rangle ~ \hat{n}_\alpha +
+        \sum_{\alpha, \beta ~ \in ~ \mathrm{active~space}} \langle \alpha \vert \hat{\mathcal{A}}
+        \vert \beta \rangle ~ \hat{c}_\alpha^\dagger\hat{c}_\beta \\
+        &&\hat B = \frac{1}{2} \left\{ \sum_{\alpha, \beta \leq 2N_\mathrm{docc}}
+        \langle \alpha, \beta \vert \hat{\mathcal{B}} \vert \beta, \alpha \rangle
+        ~ \hat{n}_\alpha \hat{n}_\beta + \sum_{\alpha, \beta, \gamma, \delta ~
+        \in ~ \mathrm{active~space}} \langle \alpha, \beta \vert \hat{\mathcal{B}}
+        \vert \gamma, \delta \rangle ~ \hat{c}_{\alpha}^\dagger \hat{c}_{\beta}^\dagger
+        \hat{c}_{\gamma} \hat{c}_{\delta} \right\}.
+
+    In the latter equations :math:`N_\mathrm{docc}` denotes the doubly-occupied orbitals,
+    if any, not included in the active space and
+    :math:`\langle \alpha \vert \hat{\mathcal{A}} \vert \beta \rangle` and
+    :math:`\langle \alpha, \beta \vert\hat{\mathcal{B}} \vert \gamma, \delta \rangle`
+    are the matrix elements of the one- and two-particle operators
+    :math:`\hat{\mathcal{A}}` and :math:`\hat{\mathcal{B}}`, respectively.
+
+    The function utilizes tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
+    to build the second-quantized operator and map it to basis of Pauli matrices via the
+    Jordan-Wigner or Bravyi-Kitaev transformation. Finally, the qubit operator is
+    converted to a a PennyLane observable by the function :func:`~.convert_observable`.
+
+    Args:
+        me_table (array[float]): Numpy array with the table of matrix elements.
+            For a single-particle operator this array will have shape
+            ``(me_table.shape[0], 3)`` with each row containing the indices
+            :math:`\alpha`, :math:`\beta` and the matrix element :math:`\langle \alpha \vert
+            \hat{\mathcal{A}}\vert \beta \rangle`. For a two-particle operator this
+            array will have shape ``(me_table.shape[0], 5)`` with each row containing
+            the indices :math:`\alpha`, :math:`\beta`, :math:`\gamma`, :math:`\delta` and
+            the matrix elements :math:`\langle \alpha, \beta \vert \hat{\mathcal{B}}
+            \vert \gamma, \delta \rangle`.
+        init_term: the contribution of doubly-occupied orbitals, if any, or other quantity
+            required to initialize the many-body observable.
+        mapping (str): specifies the fermion-to-qubit mapping. Input values can
+            be ``'jordan_wigner'`` or ``'bravyi_kitaev'``.
+
+    Returns:
+        pennylane.Hamiltonian: the fermionic-to-qubit transformed observable
+
+    **Example**
+
+    >>> s2_matrix_elements, init_term = get_spin2_matrix_elements('h2', './pyscf/sto-3g')
+    >>> s2_obs = observable(s2_matrix_elements, init_term=init_term)
+    >>> print(type(s2_obs))
```

Comment:
I think this can be removed. We know by construction it should be a PL observable, right? I think there's no need to check that in example code